### ISSUE TITLE
Add an option to hide the status bar until video is playing

### ISSUE THREAD

**Issue by blrichwine** [ableplayer/ableplayer#151]
Some of the feedback we have received on the ableplayer is that the player's appearance looks too chunky or not modern enough. 

The opinion changed and the objections went away when we created an implementation where we hid the status bar by default (so there are only 2 lines of controls displayed under the poster image initially). The status bar appears (and stays visible) once a user plays the video for the first time. 

You can see an example of this at:
https://atac.iu.edu/

Note that the status bar is hidden by default, and appears only once the video is played.

**Comment by terrill** [comment]
Interesting idea @blrichwine. Should the appearance of the status bar be triggered by other events too though, not just play? For example, if the user scrubs ahead before playing, or changes the playback rate, it would be nice for them to see the results of those actions on the status bar.

**Comment by blrichwine** [comment]
Yes. I'd suppose once almost any interaction takes place, calling a method to reveal the status bar makes sense.

**Comment by blrichwine** [comment]
Oh, one other detail... in our implementation we didn't hide it with a display:none, or aria-hidden, etc.

We simply clipped it so it was visibly hidden... non-visual users will still have access to the status bar. That way when they are initially exploring the player, the status bar is present and they will know it is available to them.

**Comment by terrill** [comment]
Also @blrichwine there's another related issue in the queue to hide the control bar as well (see issue #119). My most recent proposal was to add this feature as a user preference, but turn it off by default and allow users who are bothered by visible controls to enable it. Do you think that approach would satisfy your needs related to the status bar, or is that really a separate issue?

**Comment by blrichwine** [comment]
My 2 cents would be to have it a separate option. Mostly, because I like having available input controls visible and obvious. 

When the video isn't being played or interacted with, the status info isn't so useful.
